### PR TITLE
refactor: Programmatic init FilelistViewController

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**

### CODE REVIEW

File: kDrive/UI/View/Header view/FilesHeaderView.swift [hunk 1]
```diff
@@ -73,6 +74,10 @@ class FilesHeaderView: UICollectionReusableView {
         )
         let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(didTapOnCard))
         uploadCardView.addGestureRecognizer(tapGestureRecognizer)
+
+        selectView = SelectView.instantiate()
```
Comment (adrien-coye) [inline kDrive/UI/View/Header view/FilesHeaderView.swift:78]: So I guess your intent is to keep the `Class.instantiate()` pattern only for xib related views ?

**Review by adrien-coye — COMMENTED** [review]
I read the code, all good. I'll play with it tomorrow.

File: kDrive/UI/View/Header view/FilesHeaderView.swift (hunk 1 above)
Comment (PhilippeWeidmann) [reply kDrive/UI/View/Header view/FilesHeaderView.swift:78]: Yes for now I didn't to touch the XIBs 👍

**Review by adrien-coye — APPROVED** [review]
I did play a bit with it. All good.

**Pull request merged**